controllers/placementapi_controller.go

```diff
@@ -338,6 +407,39 @@ func (r *PlacementAPIReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
+func (r *PlacementAPIReconciler) findObjectsForSrc(src client.Object) []reconcile.Request {
```

Comment:
This should be added as a Watch in via the NewControllerManagedBy(mgr) builder otherwise the TLS secrets are not watched.

```golang
		Watches(
			&source.Kind{Type: &corev1.Secret{}},
			handler.EnqueueRequestsFromMapFunc(r.findObjectsForSrc),
			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{}),
		).
```

Reply:
Thanks a lot for raising this and the example, I added it to the other tests.

Reply:
Thanks!